.ci/cico_build_and_deploy_custom_rh-che.sh

```diff
@@ -11,28 +11,21 @@ set +o nounset
 
 /usr/sbin/setenforce 0
 
-export CHE_DOCKER_BASE_IMAGE=eclipse/che-server:nightly-centos
+export RH_CHE_AUTOMATION_SERVER_DEPLOYMENT_URL=https://rhche-che6-automated.dev.rdu2c.fabric8.io/
+export BASEDIR=$(pwd)
 export ORIGIN_CLIENTS_URL=http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/origin-clients-3.7.1-2.el7.x86_64.rpm
-export OPENSHIFT_RDU2C_URL=https://dev.rdu2c.fabric8.io:8443/
+export OSD_RDU2C_URL=https://dev.rdu2c.fabric8.io:8443/
```

Comment:
I would prefer to call it `DEV_CLUSTER_URL`

Reply:
done.